### ISSUE TITLE
a11y unit tests documentation should be more discoverable

### ISSUE THREAD

**Issue by Blackbaud-IsaacAggrey** [blackbaud/skyux2-docs#358]
If you start at the [SKY UX Learn page](https://developer.blackbaud.com/skyux/learn) - you'll see the big "Accesibility" section and probably end up at ["Test for Accessibility"](https://developer.blackbaud.com/skyux/learn/accessibility/test) but [that section](https://developer.blackbaud.com/skyux/learn/accessibility/test#run-automated-accessibility-tests) links to the [discouraged e2e test](https://developer.blackbaud.com/skyux/learn/get-started/advanced/accessibility-analyzer) instead of the [accessibility unit tests](https://developer.blackbaud.com/skyux/learn/get-started/advanced/accessibility-unit-tests) section.

The documentation and link should be updated to guide the user to the a11y unit tests.

**Comment by Blackbaud-DeniseCisneros** [comment]
Merged in https://github.com/blackbaud/skyux2-docs/pull/361
